ui/helpers/utils/util.js

```diff
@@ -593,7 +593,8 @@ export function getAssetImageURL(image, ipfsGateway) {
     // In the future, we can look into solving the root cause, which might require
     // no longer using multiform's CID.parse() method within the assets-controller
     try {
-      return getFormattedIpfsUrl(ipfsGateway, image, true);
+      console.log('heeeereeee', image, ipfsGateway);
+      return await getFormattedIpfsUrl(ipfsGateway, image, true);
```

Comment:
Updated this because `getFormattedIpfsUrl` is now async

Reply:
Prob want to remove that `console.log`?